CHANGELOG.md

```diff
@@ -26,73 +26,59 @@
+- Fixed crash in `rescript build` on Windows [#5516](https://github.com/rescript-lang/rescript-compiler/pull/5516)
+- Fixed `rescript init` command not working [#5526](https://github.com/rescript-lang/rescript-compiler/pull/5526)
+- Fix issue with compiler log not terminated that causes problems with editor extension not clearing issues when fixed [#5545](https://github.com/rescript-lang/rescript-compiler/issues/5545)
 
-#### :house: [Internal]
+##### :nail_care: Polish
+
+- Changed Linux build to depend on GLIBC 2.28 again for compatibility with Debian 10.
 
 - Proper M1 support (CI now supports M1 native builds)
 
 **Syntax**
 
 #### :boom: Breaking Change
 
-- Remove parsing of "import" and "export" which was never officially supported https://github.com/rescript-lang/syntax/pull/597 https://github.com/rescript-lang/syntax/pull/599
-  - Example: `export type t = int`
-  - Fix: `@genType type t = int`
-  - Example2: `import realValue: complexNumber => float from "./MyMath"`
-  - Fix2: `@genType.import("./MyMath") external realValue: complexNumber => float = "realValue"`
+- `@bs.send.pipe` is now removed. Earlier it was deprecated.
+- Missing labels in function application is now an error (https://forum.rescript-lang.org/t/ann-more-strict-checks-in-missed-labels/2117).
+  - Example: `let f = (x, ~z) => x + z; f(1, 2)`
+  - Fix: do `let f = (x, ~z) => x + z; f(1, ~z=2)` instead
+- Externals without `@val` annotations do not work anymore, and externals with `= ""` give an error.
+  - Example: `external setTimeout: (unit => unit, int) => float = "setTimeout"` is not supported anymore.
+  - Fix: use `[@val] external setTimeout: (unit => unit, int) => float = "setTimeout"` instead.
+  - Example2: `[@val] external setTimeout: (unit => unit, int) => float = ""` is not supported anymore.
+  - Fix2: use `[@val] external setTimeout: (unit => unit, int) => float = "setTimeout"` instead.
+- String constants don't need escaping anymore.
+  - Example: `let blockCommentsRe = %re("/\\/\\*([^*]|[\\r\\n]|(\\*+([^*/]|[\\r\\n])))*\\*+\\//g")`.
+  - Fix: use `let blockCommentsRe = %re("/\/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+\//g")` instead.
+- Externals without `@val` annotations do not work anymore, and externals with `= ""` give an error.
+  - Example: `external setTimeout: (unit => unit, int) => float = "setTimeout"` is not supported anymore.
+  - Fix: use `[@val] external setTimeout: (unit => unit, int) => float = "setTimeout"` instead.
+  - Example2: `[@val] external setTimeout: (unit => unit, int) => float = ""` is not supported anymore.
+  - Fix2: use `[@val] external setTimeout: (unit => unit, int) => float = "setTimeout"` instead.
+- String constants don't need escaping anymore.
+  - Example: `let blockCommentsRe = %re("/\\/\\*([^*]|[\\r\\n]|(\\*+([^*/]|[\\r\\n])))*\\*+\\//g")`.
+  - Fix: use `let blockCommentsRe = %re("/\/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+\//g")` instead.
```

Comment:
Lines were duplicated by accident, and I've reworded the escaping entry
```suggestion
- Strings processed at compile-time don't need escaping anymore.
  - Example:
    ```
    let blockCommentsRe = %re("/\\/\\*([^*]|[\\r\\n]|(\\*+([^*/]|[\\r\\n])))*\\*+\\//g")
    let escapedInterpolated = `invalid string:\\t ${s}`
    ```
  - Fix:
    ```
    let blockCommentsRe = %re("/\/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+\//g")`
    let escapedInterpolated = `invalid string:\t ${s}`
    ```
```

Reply:
Why "processed"?
What about normal string?

Reply:
They already didn't need escaping.

In v9:
* `"\\t"` generates literal `"\\t"`
* `"\t"` generates literal `"\t"`
*  ``` `\\t` ``` generates `"\t"`
*  ``` `\t` ``` also generates literal `"\t"`

In v10 the third option was removed, so everything is literal even in backtick strings. I figure calling them "processed" strings is easiest to describe, because `%re()` doesn't require a backtick string but did process them in that fashion.